src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+const link = document.querySelectorAll('.list-item__link');
+
+link.forEach((elem) => {
+  elem.addEventListener('click', (event) => {
+    event.preventDefault();
+  });
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  const bigImg = document.querySelector('#largeImg');
+  const images = document.querySelectorAll('.gallery__img.gallery__thumb');
+
+  images.forEach((image) => {
+    image.addEventListener('click', () => {
+      const srcImg = image.getAttribute('src');
+      const indexThumb = srcImg.indexOf('-thumb');
+      const newSrc = 'images' + srcImg.slice(0, indexThumb) + '.png';
```

Comment:
There is an issue with constructing the new image source URL. The 'slice' method is used incorrectly here. The 'srcImg.slice(0, indexThumb)' will include the 'images' directory in the path, which is incorrect. You should ensure that the new source path is constructed correctly by adjusting the slicing logic.